test/e2e/scheduling/preemption.go

```diff
@@ -178,14 +178,14 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 
 		ginkgo.By("Run a high priority pod that has same requirements as that of lower priority pod")
 		// Create a high priority pod and make sure it is scheduled on the same node as the low priority pod.
-		runPausePod(f, pausePodConfig{
+		runPausePodWithTimeout(f, pausePodConfig{
 			Name:              "preemptor-pod",
 			PriorityClassName: highPriorityClassName,
 			Resources: &v1.ResourceRequirements{
 				Requests: podRes,
 				Limits:   podRes,
 			},
-		})
+		}, framework.PodReadyBeforeTimeout)
```

Comment:
do not reuse the constant if the action doesn't correlate with it's description.

Reply:
How about reuse `PodStartTimeout`?

Reply:
`PodStartTimeout` seems like the appropriate one based on its comment

Reply:
or `PodStartShortTimeout`. 5 minutes seems excessive to me, as it might hide backoff/retry errors.

Reply:
`PodStartShortTimeout` sounds good. Let me update and update the title.